Cargo.lock

```diff
@@ -1,14 +1,3 @@
-[root]
```

Comment:
Not sure if it matters, but running `cargo build` on the latest nightly (11-07) seems to shuffle things around a bit in the lockfile.

Reply:
I'll push a commit to `master` that makes this change so we avoid cluttering the PR

Reply:
Done in 3188e79991563a0e6fa35fb96c05986303ee48d1, so if you rebase you should get it.

Reply:
Thanks, done!